deebot_client/messages/__init__.py

```diff
@@ -53,7 +55,7 @@ def get_message(message_name: str, data_type: DataType) -> type[Message] | None:
     If there exists no exact match, some conversations are performed on the name to get message object similar to the name.
     """
     messages = MESSAGES.get(data_type)
-    if messages is None:
+    if messages is None:  # pragma: no cover
```

Comment:
```suggestion
    if messages is None:
```

Never exclude code from the coverage report